src/ctia/http/handler.clj

```diff
@@ -179,6 +185,12 @@
                     :tokenUrl token-url
                     :flow flow}))))
 
+(defn- mark-disabled-entities
+  [{{:keys [enabled?]} :FeaturesService}
+   {:keys [entity] :as entity-map}]
+  (cond-> entity-map
+    (not (enabled? entity)) (assoc :no-api? true)))
```

Comment:
We must preserve the hard coded value for [identity](https://github.com/threatgrid/ctia/blob/f40375ae1b690c6cf298d6cfac012c51e2134652/src/ctia/entity/identity.clj#L101), it should never be exposed in the API (and thus not be configurable). Then I would rather use update (I cannot argue, just a matter of taste :-) ):
```suggestion
 (update entity-map
             :no-api?
            #(or % (not (enabled? entity))))
```
btw, in the previous PR you had a `disabled?` counterpart that would be handy here :-)
WDYT?

Reply:
I juste saw  @vemv opposite feedback on `enabled`/`disabled` :-)
It's a minor remark, you can ignore it but I still suggest the rest of the comment :-)

Reply:
But `cond->` keeps the entity map intact, and only adds `:no-api?` if there's a record in the config. If there's no such record, it just passes the map through, either it has `:no-api?` already or it doesn't. So `ctia.entity.identity` should be fine.